### PR TITLE
Web: Setup Mantine UI library

### PR DESCRIPTION
**Pull request by aleksays**

### CODE REVIEW

File: services/web/src/pages/_app/index.jsx [hunk 1]
```diff
@@ -15,13 +18,21 @@ const App = ({ Component, pageProps }) => (
...
       <title>Ship</title>
     </Head>
-    <QueryClientProvider client={queryClient}>
-      <PageConfig>
-        <Component {...pageProps} />
-      </PageConfig>
-      <ReactQueryDevtools position="bottom-right" />
-    </QueryClientProvider>
-    <Toaster />
+    <MantineProvider
```
Comment (Oigen43) [inline services/web/src/pages/_app/index.jsx:21]: let's move QueryClientProvider upper MantineProvider

**Review by Oigen43 — CHANGES_REQUESTED** [review]

**Comment by Oigen43** [comment]
Nice, let's continue integrating Mantine UI in this PR

File: services/web/src/pages/sign-in/index.page.jsx [hunk 2]
```diff
@@ -37,25 +38,31 @@ const SignIn = () => {
           onSubmit={handleSubmit(onSubmit)}
           className={styles.form}
         >
-          <Input
-            name="email"
+          <TextInput
+            {...register('email')}
             label="Email Address"
+            labelProps={{
+              'aria-invalid': errors.email && 'true',
```
Comment (Oigen43) [inline services/web/src/pages/sign-in/index.page.jsx:45]: do we really need `&& 'true'` 🤔

File: services/web/src/pages/sign-in/index.page.jsx [hunk 3]
```diff
@@ -37,25 +38,31 @@ const SignIn = () => {
...
+            }}
             placeholder="Email"
-            control={control}
-            error={errors.email}
+            error={errors?.email?.message}
```
Comment (Oigen43) [inline services/web/src/pages/sign-in/index.page.jsx:48]: Can we map errors to user them in next format?
`error={errors.email}`

File: services/web/src/pages/sign-in/index.page.jsx [hunk 4]
```diff
@@ -37,25 +38,31 @@ const SignIn = () => {
...
           />
-          <Input
-            name="password"
-            type="password"
+          <PasswordInput
+            {...register('password')}
             label="Password"
+            labelProps={{
+              'aria-invalid': errors.password && 'true',
```
Comment (Oigen43) [inline services/web/src/pages/sign-in/index.page.jsx:54]: do we really need && 'true' 🤔

**Review by Oigen43 — CHANGES_REQUESTED** [review]

File: services/web/src/pages/sign-in/index.page.jsx (hunk 2 above)
Comment (aleksays) [reply services/web/src/pages/sign-in/index.page.jsx:45]: Yep, cuz it's html attr value — it should be string.

**Pull request merged**